internal/backend/crypto/gpg/cli/loader.go

```diff
@@ -23,8 +24,9 @@ type loader struct{}
 func (l loader) New(ctx context.Context) (backend.Crypto, error) {
 	debug.Log("Using Crypto Backend: %s", name)
 	return New(ctx, Config{
-		Umask: fsutil.Umask(),
-		Args:  GPGOpts(),
+		Umask:  fsutil.Umask(),
+		Args:   GPGOpts(),
+		Binary: os.Getenv("GOPASS_GPG_BINARY"),
```

Comment:
Where is that used? Does it really work??

Reply:
The loader? Yes, that's being invoked through the blank imports.